tests/Unit/Evolution/Systems/NewtonianEuler/Test_Characteristics.cpp

```diff
@@ -318,6 +318,14 @@ void test_right_and_left_eigenvectors() noexcept {
   test_left_and_right_eigenvectors_impl(
       density, velocity, specific_internal_energy, unit_normal, ideal_gas_eos);
 
+  // Test also in case where unit normal is aligned to axes
+  for (size_t i = 0; i < Dim; ++i) {
```

Comment:
Does the axis-aligned test also work for the polytrope?

Reply:
Yes. Pushed update.